src/bindActionCreators.js

```diff
@@ -1,5 +1,7 @@
 function bindActionCreator(actionCreator, dispatch) {
-  return (...args) => dispatch(actionCreator(...args))
+  const boundActionCreator = (...args) => dispatch(actionCreator(...args))
+  boundActionCreator.__boundContext__ = dispatch
```

Comment:
Is there a reason we're referencing `dispatch` instead of just setting a `__bound__` property to `true`?